frontend/packages/employee-frontend/src/api/attendances.ts

```diff
@@ -201,13 +201,26 @@ export async function postDeparture(
     .catch((e) => Failure.fromError(e))
 }
 
+function compareFirstNames(
+  a: JsonOfObject<AttendanceChild>,
+  b: JsonOfObject<AttendanceChild>
+) {
+  if (a.firstName < b.firstName) {
+    return -1
+  }
+  if (a.firstName > b.firstName) {
+    return 1
+  }
+  return 0
+}
+
 function deserializeAttendanceResponse(
   data: JsonOf<AttendanceResponse>
 ): AttendanceResponse {
   {
     return {
       unit: data.unit,
-      children: data.children.map((attendanceChild) => {
+      children: data.children.sort(compareFirstNames).map((attendanceChild) => {
```

Comment:
Now two children with same first name will have undefined order, wouldn't it make more sense to then sort by last name next? Also a custom comparator for such a simple case is quite verbose compared to using lodash.